src/platform-includes/performance/traces-sample-rate/javascript.mdx

```diff
@@ -2,6 +2,6 @@
 Sentry.init({
   // ...
 
-  tracesSampleRate: 0.2,
+  tracesSampleRate: 0.5,
```

Comment:
l: Just noticed that in the [text above this include](https://sentry-docs-git-abhi-sample-rate-js.sentry.dev/platforms/javascript/configuration/sampling/#setting-a-uniform-sample-rate) we still talk about 0.2. I think we should update this as well.

Reply:
Done!